node/resolvers/checkout/orderFormItem.ts

```diff
@@ -16,5 +16,8 @@ export const resolvers = {
     listPrice: ({ listPrice }: Params) => listPrice / 100,
     price: ({ price }: Params) => price / 100,
     sellingPrice: ({ sellingPrice }: Params) => sellingPrice / 100,
+    productCategoryIds: ({ productCategoryIds }: Params) => productCategoryIds,
```

Comment:
please, remove lines 19 to 21, since they are not necessary

Reply:
Sure, done!